stm32-metapac/build.rs

```diff
@@ -131,6 +155,14 @@ fn main() {
         peripherals: Vec::new(),
     };
 
+    // Load RCC register for chip
+    let chip_family = chip.family.to_ascii_lowercase();
+    let rcc_family = chip_family.strip_prefix("stm32").unwrap();
```

Comment:
It's nicer to get the RCC version from `chip.peripherals` instead of doing string operations on the family. This may not always be correct if there are exceptions.

Reply:
This explodes if the chip doesn't have RCC right? currently stm32-metapac allows using all the chips, it's only embassy-stm32 that only supports certain families. We should try keeping the metapac building with all chips

Reply:
Yeah, thats a good point. I thought RCC was always there. I'll change it to look for the RCC peripheral, and Option'ize the rcc register find and clock lookup.

Reply:
yeah, all chips have it. I meant we don't have the regs yaml for all chips yet. For these the metapac should still build, even if incomplete.